ci/package_list.sh

```diff
@@ -15,29 +15,5 @@ export RELEASE_PACKAGES
 # Deprecated packages are still built but not explicitly tested
 # This allows users to still access the packages if needed, however
 # it is only reserved for packages with a more public facing API
-DEPRECATED_PACKAGES=(
-    c8y-configuration-plugin
-)
+DEPRECATED_PACKAGES=()
```

Comment:
yeah I figured there is no harm in keeping it for now...as we might be able to re-think how the linux packages are organized since we have a multi-call binary...